src/tribler/core/components/tag/db/tag_db.py

```diff
@@ -19,6 +18,32 @@
 HIDE_THRESHOLD = -2
 
 
+class Operation(IntEnum):
+    ADD = 1
+    REMOVE = 2
+
+
+class Predicate(IntEnum):
+    HAS_CONTRIBUTOR = 1
```

Comment:
I suggest to remove the `HAS_` prefix from these names to make it better reflect the relation. With the `HAS_` prefix, it looks to me like the predicate indicates a boolean statement (which, strictly speaking, a predicate is though). But in the context of Tribler, a triple is not a boolean statement.

Reply:
Done